cfstore/scripts/aggregatebmetadata.py

```diff
@@ -36,27 +15,13 @@ def manage_types(value):
             with open("{{homedir}}"+filename+"_variables_bmetadata.json","w") as writepath:
```

Comment:
f-strings are preferable, in general:
```suggestion
            with open(f"{{{{homedir}}}}{filename}_variables_bmetadata.json","w") as writepath:
```